### PR TITLE
Update worldData.js

### PR DESCRIPTION
**Pull request by nehasangeetajha**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ import palak from "../../photos/pal-16.jpg";
 import AkshaySharma008 from "../../photos/AkshaySharma008.jpg";
 import AdityaBirangal from "../../photos/AdityaBirangal.jpg";
 import ikoala21 from "../../photos/ikoala21.jpg";
+import sangeetajha from "../../photos/SangeetaJha.png";
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:45]:
```suggestion
import sangeetajha from "../../photos/SangeetaJha.jpg";
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@nehasangeetajha Please add the picture to this PR itself and close #86. Also, it should be a `jpg` file. Please follow the [guidelines mentioned](https://github.com/abhinavsri360/GDSC-Lead-Map/blob/main/CONTRIBUTING.md).

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
Data Looks good to me, do add our picture in this PR itself 😄

**Comment by nehasangeetajha** [comment]
Done!

**Review by aravindvnair99 — APPROVED** [review]

**Review by abhinavsri360 — APPROVED** [review]
Thanks for adding yourself.

**Pull request merged**